<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtTcpSeverClass</class>
 <widget class="QWidget" name="qtTcpSeverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtTcpSever</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Recv">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Send">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>258</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Port：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Port"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Listen">
      <property name="text">
       <string>侦听</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtTcpSever.qrc"/>
 </resources>
 <connections/>
</ui>
